### PR TITLE
[FEATURE] Add a basic content element

### PR DESCRIPTION
**Pull request by linawolf**
Demonstrates the following features:

Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/801
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1013

### CODE REVIEW

File: Configuration/TCA/Overrides/tt_content_basic_content.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
```
Comment (brotkrueml) [inline Configuration/TCA/Overrides/tt_content_basic_content.php:1]: Shouldn't we also add an entry to the New Content Element Wizard as best practise?

```php
ExtensionManagementUtility::addPlugin(
    [
        'label' => 'Example - basic content',
        'description' => 'Content element as demonstration for basic content',
        'group' => 'default',
        'value' => $key,
        'icon' => 'content-text',
    ],
    'CType',
    'examples,
);
```

see also https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1012

Comment (linawolf) [reply Configuration/TCA/Overrides/tt_content_basic_content.php:1]: This is not a plugin but a plain content element with some TypoScript. And with https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/13.0/Feature-102834-Auto-registrationOfNewContentElementWizardViaTCA.html they are automatically listed when created  via ExtensionManagementUtility::addTcaSelectItem. Try it out, it is working

Comment (brotkrueml) [reply Configuration/TCA/Overrides/tt_content_basic_content.php:1]: Yup, that's why "CType" is used (and not plugin)....

But, is description and the icon added? Maybe then also add it to the addTcaSelectItem if this works, I am fine.

Comment (linawolf) [reply Configuration/TCA/Overrides/tt_content_basic_content.php:1]: yes they are added

**Review by froemken — APPROVED** [review]

**Pull request merged**